2_observations.yml

```diff
@@ -134,3 +135,20 @@ targets:
       mgd_to_cms = 0.0438, cfs_to_cms = 0.0283,
       mg_to_cm = 4404.88, ft_to_m = 0.3048)
 
+ # reservoir surface area data
+ # you must download reservoir Monthly Shapefiles from here: 'http://umnlcc.cs.umn.edu/realsat/'
+ # unzip downloaded folder to 2_observations/in
+ # such that the structure becomes
+ # 2_observations/in/ID-Number
+      # in/573567/573567
+      # in/009130/009130
+  2_observations/out/reserviors_sf_directory:
+    command: read_monthly_surface_area_data(res_sf_dir = target_name)
+
+  2_observations/out/realsat_monthly_surface_area.csv:
+    command: combine_reaLSAT_reservoir_data(
```

Comment:
If you want to use capital letters in `reaLSAT` (which is OK with me), let's use them consistently throughout and exactly as presented by Ankush's team: in the parent folder it'd become `2_observations/in/ReaLSAT` and here it'd be `combine_ReaLSAT_reservoir_data()`. Or just use entirely lowercase throughout.